index.js

```diff
@@ -10,6 +10,7 @@ var resolveGlob = require('to-absolute-glob');
 var globParent = require('glob-parent');
 var path = require('path');
 var extend = require('extend');
+var sepRe = (process.platform === 'win32' ? /[\/\\]/ : /\/+/);
```

Comment:
For `win32` it's checking for either `/` or `\` and replacing with the `path.sep`. This is because the user might have passed in `/` on windows in their glob pattern.